### ISSUE TITLE
Output

### ISSUE THREAD

**Issue by michaeloc** [liuyiding1993/ICDE2020_GMVSAE#10]
Hi, could you help me? I'm try to get the output from decoder in order to compare with the targets, but I don't have success. I'm trying a solution as  follows:

results = tf.matmul(outputs, tf.transpose(self.out_w))
results = tf.nn.bias_add(logits, self.out_b)     

However, when try to compare "results == targets" the results are very different. I'm doing something wrong because the scores are very good.

**Comment by BruceLuke** [comment]
A good question
